### DISCUSSION TITLE
iOS feature request. IAC Driver implementation.

### DISCUSSION THREAD

**Discussion by pereshaped** [plugdata-team/plugdata#1861] [Ideas]
Is there any chance that the iOs IAC driver could be implemented as an option for audio and MIDI in Plug Data?  I know that it is a legacy driver, but it's still valid. Some audio apps have it and it is very handy. It allows audio and MIDI apps to talk to each other without having to use third party apps.

**Comment by timothyschoen** [comment]
Yep, I was planning to do this :)

The last version of the iOS app got held up in Apple's review process, because they didn't see the use of the background audio feature. And I think they are right: we need to also have IAC for that to be useful. So now I think I might as well finish off the next version, add IAC and try again if we can get through the review process.

**Comment by pereshaped** [comment]
That's fantastic news! Thank you

**Comment by pereshaped** [comment]
Amazing how they (Apple) control the process!
